release_build_files/readme.md

```diff
@@ -571,6 +571,8 @@ code.
 -   Changes
     -   Firestore/Database (Desktop): Upgrade LevelDb dependency to 1.23
         ([#886](https://github.com/firebase/firebase-cpp-sdk/pull/886)).
+    -   Firestore (desktop): Enabled Snappy compression support in LevelDb
```

Comment:
Is this enabled in RTDB now as well? (I noticed the linker error happened in the database integration test too, so it may be using it in both?

Reply:
(also - "**D**esktop")

Reply:
Yes. In cmake, if Firestore is included in the build then both RTDB and Firestore share the LevelDb library from Firestore. However, if Firestore is excluded from the build then RTDB will use a LevelDb without Snappy support. We should probably do something about this. We had a meeting yesterday where it sounded like a good idea might be to unconditionally download the firebase_ios_sdk and use LevelDb from it regardless of whether or not Firestore is being used. Would you like to hold off merging this until we figure that out?

Reply:
Oh wait, I see you were pointing out that the release notes entry needed a correction. I've updated it in 8bb6bf51cf150519894afae0c6ffb61e1e37826a.